sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/models/UpdateRoomOptions.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.rooms.models;
+
+import java.time.OffsetDateTime;
+
+/**
+ * The options for updating a room.
+ */
+public final class UpdateRoomOptions {
+
+    /**
+     * The starting time point of the room.
```

Comment:
Same here, let's make the descriptions better and consistent with what gets generated

Reply:
added, updated all comments